source/graphics/GridGraphic.js

```diff
@@ -50,11 +99,69 @@ export default class GridGraphic extends Graphic {
     }
   }
 
+  _onMarqueeMouseUp(event) {
+    if (this._makingMarqueeSelection) {
+      const marqueeBounds = {
+        x1: Math.min(this._marqueeStart.x, this._mouseAt.x),
+        x2: Math.max(this._marqueeStart.x, this._mouseAt.x),
+        y1: Math.min(this._marqueeStart.y, this._mouseAt.y),
+        y2: Math.max(this._marqueeStart.y, this._mouseAt.y),
+
+      }
+      this._selectedTiles = this._tiles.filter((tile) => {
+        const center = hexagonGrid.tileCenterPoint(tile.position)
+        center.x /= 2
+        center.y /= 2
+        return center.x > marqueeBounds.x1 && center.x < marqueeBounds.x2 &&
+          center.y > marqueeBounds.y1 && center.y < marqueeBounds.y2
+      })
+      this._makingMarqueeSelection = false
+    } else if (this._draggingMultiSelect) {
+      const offset = {
+        x: event.offsetX - this._draggingMultiSelectOrigin.x,
+        y: event.offsetY - this._draggingMultiSelectOrigin.y,
+      }
+
+      let noOverlaps = true
+      // assign `newPosition` to each tile
+      this._selectedTiles.forEach((tile) => {
```

Comment:
Could use `Array.some()`, depending on our browser support requirements. It bails early from loop!